pkg/crc/services/dns/dns.go

```diff
@@ -40,6 +41,26 @@ func RunPostStart(serviceConfig services.ServicePostStartConfig) (services.Servi
 	// Remove the dnsmasq container if it exists during the VM stop cycle
 	_, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq")
 
+	// Add a virtual dummy network interface and set the IP as `192.168.126.11` which is the IP used to create the bundle
+	if _, err := serviceConfig.SSHRunner.Run("sudo ip link add eth10 type dummy && sudo ip addr add 192.168.126.11/24 dev eth10"); err != nil {
+		result.Success = false
+		result.Error = err.Error()
+		return *result, err
+	}
+	// Todo: Move those to snc script when start building for 4.4 bundles
+	// Add node-ip field in kublet service and set it to the static IP `192.168.126.11`
+	if _, err := serviceConfig.SSHRunner.Run(`sudo sed -i.back '/kubelet /a\      \--node-ip=192.168.126.11 \\' /etc/systemd/system/kubelet.service`); err != nil {
+		result.Success = false
+		result.Error = err.Error()
+		return *result, err
+	}
+	// Reload the kubelet service since the service changed happen
+	if _, err = serviceConfig.SSHRunner.Run("sudo systemctl daemon-reload"); err != nil {
```

Comment:
This is calling "daemon-reload", not I'm not sure this triggers a restart of `kubelet.service` if it changed?

Reply:
@cfergeau you mean the comment right?

Reply:
Ah, I was a bit confused by the comment, but also because I did not look at the code (kubelet is not running at this point)
`Reload the kubelet systemd unit file since it was modified`?

Reply:
Isn't this going to be adding a `--node-ip=192.168.126.11` line each time we run `crc start` though?